odoo/tools/sql.py

```diff
@@ -123,12 +123,20 @@ def constraint_definition(cr, constraintname):
     cr.execute("SELECT pg_get_constraintdef(oid) FROM pg_constraint WHERE conname=%s", (constraintname,))
     return cr.fetchone()[0] if cr.rowcount else None
 
+def constraint_comment(cr, constraintname):
+    """ Return the given constraint's comment. """
+    cr.execute("SELECT description FROM pg_description d JOIN pg_constraint c "
+               "ON c.oid = d.objoid AND conname = %s;", (constraintname,))
```

Comment:
Constraint names aren't unique, so even if the framework prefixes them with the model name, may I suggest we fully qualify them with the table?

```python
query = """
    SELECT description FROM pg_description d
                       JOIN pg_constraint c ON (c.oid = d.objoid AND conname = %s)
                       JOIN pg_class t ON (t.oid = c.conrelid AND t.relname = %s)
"""
cr.execute(query, (constraintname, tablename))
```
(requires passing tablename)

We should probably do the same in `constraint_definition()`, etc.